src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-details/rbd-details.component.html

```diff
@@ -143,6 +143,17 @@
         <cd-rbd-configuration-table [data]="selection['configuration']"></cd-rbd-configuration-table>
       </ng-template>
     </li>
+
+    <li ngbNavItem="performance">
+      <a ngbNavLink
+         i18n>Performance</a>
+      <ng-template ngbNavContent>
+        <cd-grafana [grafanaPath]="rbdDashboardUrl"
```

Comment:
```suggestion
        <cd-grafana grafanaPath="rbd-details?var-Pool={{ selection.pool_name }}&var-Image={{ selection.name }}"
```
and removing the `onChanges()` chunk?

Reply:
I've had an issue with the linter I couldn't overcome when having that code in the template instead of the component. That's the reason I chose to put the code in the component instead.